### PR TITLE
Add clarified Sidepane documentation

### PR DESCRIPTION
**Pull request by rschwammborn**
...as proposed in #133 . Also added missing style information regarding Mapbender Templates.

### CODE REVIEW

File: de/customization/mapbender_templates.rst [hunk 1]
```diff
@@ -22,11 +22,28 @@ Regionen des Fullscreen Templates:
...
   * Footer (Fußleiste mit Impressum, Aktivitätsanzeige...)
 
-Specials: 
+Specials:
 
-  * Template mit dunklem Hintergrund 
+  * Template mit dunklem Hintergrund
   * Großansicht mit Seitenleiste
 
+Fullscreen Alternative Template
```
Comment (astroidex) [inline de/customization/mapbender_templates.rst:30]: looks like you are describing the applivcation mapbender_demo_basic and not the template.
mapbender_user_basic is only the Fullscreen teplate with some css in the css-editor to make it blue.

We should not describe the Fullscreen Alternative Template. It is not working properly, same as classic template.

File: de/customization/templates.rst [hunk 2]
```diff
@@ -138,14 +138,30 @@ In der Template-Datei wir der Name des Templates, die Regionen die angelegt werd
 Hinzufügen von Eigenschaften für die Sidepane
 *********************************************
 
-Für die Sidepane können die Eigenschaften für "Tabs" oder "Accordion" definiert werden.
+Innerhalb der Sidepane können Elemente in den Ansichten "Accordion, "Buttons" oder "None" angezeigt werden werden.
```
Comment (astroidex) [inline de/customization/templates.rst:141]: vielleiczt besser:
..können Elemente in unterschiedlichen Ansichten angezeigt werden:

File: de/customization/templates.rst [hunk 3]
```diff
@@ -138,14 +138,30 @@ In der Template-Datei wir der Name des Templates, die Regionen die angelegt werd
...
 
-- "Tabs" Reihenfolge von Mapbender Elementen in Tabs
-- "Accordion" zeigt die Elemente in einer Accordion Liste.
+- "Accordion" zeigt alle hinzugefügten Elemente in einer Accordion-Liste:
```
Comment (astroidex) [inline de/customization/templates.rst:143]: maybe: in Reitern statt Accordion

File: de/customization/templates.rst [hunk 4]
```diff
@@ -138,14 +138,30 @@ In der Template-Datei wir der Name des Templates, die Regionen die angelegt werd
...
 
-Um diese Hinzuzufügen, definert man die Funktion "getRegionsProperties" in der Template Datei.
+.. image:: ../../../figures/sidepane_accordion.png
+     :scale: 80
+
+- "Buttons" zeigt alle hinzugefügten Elemente in einer Buttonleiste:
```
Comment (astroidex) [inline de/customization/templates.rst:148]: über Buttons

File: de/customization/templates.rst [hunk 5]
```diff
@@ -138,14 +138,30 @@ In der Template-Datei wir der Name des Templates, die Regionen die angelegt werd
...
 
-Zur Konfiguration checkt man die Option in der Administration der Anwendung.
+.. image:: ../../../figures/sidepane_buttons.png
+     :scale: 80
+
+- "None" verzichtet auf Styling-Optionen und zeigt die Elemente sofort an:
```
Comment (astroidex) [inline de/customization/templates.rst:153]: Elemente direkt und in der Reihenfolge untereinander an.

File: de/customization/templates.rst [hunk 6]
```diff
@@ -138,14 +138,30 @@ In der Template-Datei wir der Name des Templates, die Regionen die angelegt werd
...
+.. image:: ../../../figures/de/sidepane_nostyle.png
+     :scale: 80
+
+Die Ansichtsoption für die Sidepane kann im Sidepane-Bereich im Mapbender-Backend ausgewählt werden. Dazu genügt ein Klick auf den gewünschten Button:
+
+.. image:: ../../../figures/sidepane_backend.png
+     :scale: 80
+
+
+Das manuelle Hinzufügen der Sidepane ist alternativ über die Funktion "getRegionsProperties" in der Template-Datei möglich.
```
Comment (astroidex) [inline de/customization/templates.rst:164]: I would delete this sentence, as it describes how to modify the template in the code base. If this should be described here it need further information.

File: de/customization/yaml.rst [hunk 7]
```diff
@@ -137,15 +137,15 @@ Die Mailerangaben starten mit mailer (z.B. smtp oder sendmail).
...
         mailer_host:       localhost
         mailer_user:       ~
         mailer_password:   ~
-    
-    
+
+
 Ein Mailer wird für die Funktionen 'Self-Registration' und 'Passwort zurücksetzen' benötigt.
-Weitere Informationen im Kapitel :ref:`users_de`.
+Weitere Informationen (auch mit Link zur Erstkonfiguration des Swiftmailers) im Kapitel :ref:`users_de`.
```
Comment (astroidex) [inline de/customization/yaml.rst:148]: is the link missing here?

File: de/customization/yaml.rst [hunk 8]
```diff
@@ -330,7 +330,7 @@ Content
     * WMS loader
```
Comment (astroidex) [inline de/customization/yaml.rst:330]: Is this description of the demo applications usefull? see https://doc.mapbender.org/de/customization/yaml.html#mapbender-demo-map
I would skip it.

File: de/customization/yaml.rst [hunk 9]
```diff
@@ -330,7 +330,7 @@ Content
...
     * Image export
     * Print client
-    * line 
+    * line
```
Comment (astroidex) [inline de/customization/yaml.rst:333]: If it should remain in the documentation better write:
measure line
measure area

File: de/customization/mapbender_templates.rst (hunk 1 above)
Comment (rschwammborn) [reply de/customization/mapbender_templates.rst]: I removed the Fullscreen Alternative template from the documentation pages.

**Review by astroidex — APPROVED** [review]
Looks good.

**Review by astroidex — APPROVED** [review]

**Review by thzanger — APPROVED** [review]

**Pull request merged**